.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rabbitmq:3.7-management
```

Comment:
Perhaps this should be `3.8-management`? 3.7 has been out of support since October 1st, 2020 :)

Reply:
I could not find any references to `3.7-management` in rabbitmq/rabbitmq-website. Was it somewhere in the Docker image docs? Do you have a link?

Reply:
Here https://registry.hub.docker.com/_/rabbitmq/ under Enabling Plugins

Reply:
OK, submitted https://github.com/docker-library/docs/pull/1890 to the Docker Hub doc maintainers